packages/@sfx/search-plugin/src/search-plugin.ts

```diff
@@ -0,0 +1,59 @@
+import { Plugin, PluginRegistry, PluginMetadata } from '@sfx/core';
+import { BridgeConfig, BrowserBridge, Query } from 'groupby-api';
+
+/**
+ * This plugin exposes an instance of the GroupBy search client
+ */
+export default class SearchPlugin implements Plugin {
+  get metadata(): PluginMetadata {
+    return {
+      name: 'search',
+      depends: [],
+    };
+  }
+
+  /**
+   * The values that the Search Data Source plugin expose to the Core entity.
+   */
+  browserBridge: SearchPluginExposedValue;
+
+  /**
+   * Instantiates an instance of the search browserBridge plugin
+   *
+   * @param options The options to instantiate the search data source browser bridge client with.
+   */
+   constructor(options: SearchPluginOptions) {
+     const customerId = options.customerId;
+     const https = !!options.https;
+
+     if (!customerId) {
+       throw new Error('customerId is not valid');
+     }
+
+     const exposedValue: any = new BrowserBridge(customerId, https, options);
```

Comment:
We should be passing only declared options to `BrowserBridge` (i.e. without `customerId` and `https`). If you take my suggestion above, you can use `browserBridgeOptions`.

```suggestion
     const exposedValue: any = new BrowserBridge(customerId, https, browserBridgeOptions);
```